.gitignore

```diff
@@ -0,0 +1,2 @@
+test.md
```

Comment:
You did well ignoring the `test.md` file, but you'll need to delete the file as it has been committed.